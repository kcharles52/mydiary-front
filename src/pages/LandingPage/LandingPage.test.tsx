//react libraries
import * as React from "react";

//third party libraries
import { shallow } from "enzyme";

//component
import { LandingPage } from ".";

describe("Render landing page properly", () => {
  let shallowWrapper;
  
  beforeEach(() => {
    shallowWrapper = shallow(<LandingPage />);
  });
  it("render landing page ", () => {
    expect(shallowWrapper).toMatchSnapshot();
  });
});
