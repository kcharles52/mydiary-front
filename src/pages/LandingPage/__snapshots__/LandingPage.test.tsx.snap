// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render landing page properly render landing page  1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LandingPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <div
        className="pg-content"
        id="home"
      >
        <div
          className="clearfix"
        >
          <div
            className="home-col"
          >
            <div
              id="home-msg"
            >
              <h2>
                Memories Memories, sweet memories
              </h2>
              <h2>
                You can easily record your memories online now 
              </h2>
              <ul>
                <li
                  className="color1"
                >
                  To do list
                </li>
                <li
                  className="color2"
                >
                  Aniverseries
                </li>
                <li
                  className="color4"
                >
                  And many more
                </li>
              </ul>
            </div>
          </div>
          <div
            className="home-col"
          >
            <div
              className="form"
            >
              <Connect(FormBox) />
            </div>
          </div>
        </div>
      </div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="clearfix"
        >
          <div
            className="home-col"
          >
            <div
              id="home-msg"
            >
              <h2>
                Memories Memories, sweet memories
              </h2>
              <h2>
                You can easily record your memories online now 
              </h2>
              <ul>
                <li
                  className="color1"
                >
                  To do list
                </li>
                <li
                  className="color2"
                >
                  Aniverseries
                </li>
                <li
                  className="color4"
                >
                  And many more
                </li>
              </ul>
            </div>
          </div>
          <div
            className="home-col"
          >
            <div
              className="form"
            >
              <Connect(FormBox) />
            </div>
          </div>
        </div>,
        "className": "pg-content",
        "id": "home",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="home-col"
            >
              <div
                id="home-msg"
              >
                <h2>
                  Memories Memories, sweet memories
                </h2>
                <h2>
                  You can easily record your memories online now 
                </h2>
                <ul>
                  <li
                    className="color1"
                  >
                    To do list
                  </li>
                  <li
                    className="color2"
                  >
                    Aniverseries
                  </li>
                  <li
                    className="color4"
                  >
                    And many more
                  </li>
                </ul>
              </div>
            </div>,
            <div
              className="home-col"
            >
              <div
                className="form"
              >
                <Connect(FormBox) />
              </div>
            </div>,
          ],
          "className": "clearfix",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                id="home-msg"
              >
                <h2>
                  Memories Memories, sweet memories
                </h2>
                <h2>
                  You can easily record your memories online now 
                </h2>
                <ul>
                  <li
                    className="color1"
                  >
                    To do list
                  </li>
                  <li
                    className="color2"
                  >
                    Aniverseries
                  </li>
                  <li
                    className="color4"
                  >
                    And many more
                  </li>
                </ul>
              </div>,
              "className": "home-col",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2>
                    Memories Memories, sweet memories
                  </h2>,
                  <h2>
                    You can easily record your memories online now 
                  </h2>,
                  <ul>
                    <li
                      className="color1"
                    >
                      To do list
                    </li>
                    <li
                      className="color2"
                    >
                      Aniverseries
                    </li>
                    <li
                      className="color4"
                    >
                      And many more
                    </li>
                  </ul>,
                ],
                "id": "home-msg",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Memories Memories, sweet memories",
                  },
                  "ref": null,
                  "rendered": "Memories Memories, sweet memories",
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "You can easily record your memories online now ",
                  },
                  "ref": null,
                  "rendered": "You can easily record your memories online now ",
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <li
                        className="color1"
                      >
                        To do list
                      </li>,
                      <li
                        className="color2"
                      >
                        Aniverseries
                      </li>,
                      <li
                        className="color4"
                      >
                        And many more
                      </li>,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "To do list",
                        "className": "color1",
                      },
                      "ref": null,
                      "rendered": "To do list",
                      "type": "li",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "Aniverseries",
                        "className": "color2",
                      },
                      "ref": null,
                      "rendered": "Aniverseries",
                      "type": "li",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "And many more",
                        "className": "color4",
                      },
                      "ref": null,
                      "rendered": "And many more",
                      "type": "li",
                    },
                  ],
                  "type": "ul",
                },
              ],
              "type": "div",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="form"
              >
                <Connect(FormBox) />
              </div>,
              "className": "home-col",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Connect(FormBox) />,
                "className": "form",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <div
          className="pg-content"
          id="home"
        >
          <div
            className="clearfix"
          >
            <div
              className="home-col"
            >
              <div
                id="home-msg"
              >
                <h2>
                  Memories Memories, sweet memories
                </h2>
                <h2>
                  You can easily record your memories online now 
                </h2>
                <ul>
                  <li
                    className="color1"
                  >
                    To do list
                  </li>
                  <li
                    className="color2"
                  >
                    Aniverseries
                  </li>
                  <li
                    className="color4"
                  >
                    And many more
                  </li>
                </ul>
              </div>
            </div>
            <div
              className="home-col"
            >
              <div
                className="form"
              >
                <Connect(FormBox) />
              </div>
            </div>
          </div>
        </div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="clearfix"
          >
            <div
              className="home-col"
            >
              <div
                id="home-msg"
              >
                <h2>
                  Memories Memories, sweet memories
                </h2>
                <h2>
                  You can easily record your memories online now 
                </h2>
                <ul>
                  <li
                    className="color1"
                  >
                    To do list
                  </li>
                  <li
                    className="color2"
                  >
                    Aniverseries
                  </li>
                  <li
                    className="color4"
                  >
                    And many more
                  </li>
                </ul>
              </div>
            </div>
            <div
              className="home-col"
            >
              <div
                className="form"
              >
                <Connect(FormBox) />
              </div>
            </div>
          </div>,
          "className": "pg-content",
          "id": "home",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="home-col"
              >
                <div
                  id="home-msg"
                >
                  <h2>
                    Memories Memories, sweet memories
                  </h2>
                  <h2>
                    You can easily record your memories online now 
                  </h2>
                  <ul>
                    <li
                      className="color1"
                    >
                      To do list
                    </li>
                    <li
                      className="color2"
                    >
                      Aniverseries
                    </li>
                    <li
                      className="color4"
                    >
                      And many more
                    </li>
                  </ul>
                </div>
              </div>,
              <div
                className="home-col"
              >
                <div
                  className="form"
                >
                  <Connect(FormBox) />
                </div>
              </div>,
            ],
            "className": "clearfix",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  id="home-msg"
                >
                  <h2>
                    Memories Memories, sweet memories
                  </h2>
                  <h2>
                    You can easily record your memories online now 
                  </h2>
                  <ul>
                    <li
                      className="color1"
                    >
                      To do list
                    </li>
                    <li
                      className="color2"
                    >
                      Aniverseries
                    </li>
                    <li
                      className="color4"
                    >
                      And many more
                    </li>
                  </ul>
                </div>,
                "className": "home-col",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <h2>
                      Memories Memories, sweet memories
                    </h2>,
                    <h2>
                      You can easily record your memories online now 
                    </h2>,
                    <ul>
                      <li
                        className="color1"
                      >
                        To do list
                      </li>
                      <li
                        className="color2"
                      >
                        Aniverseries
                      </li>
                      <li
                        className="color4"
                      >
                        And many more
                      </li>
                    </ul>,
                  ],
                  "id": "home-msg",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "Memories Memories, sweet memories",
                    },
                    "ref": null,
                    "rendered": "Memories Memories, sweet memories",
                    "type": "h2",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "You can easily record your memories online now ",
                    },
                    "ref": null,
                    "rendered": "You can easily record your memories online now ",
                    "type": "h2",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        <li
                          className="color1"
                        >
                          To do list
                        </li>,
                        <li
                          className="color2"
                        >
                          Aniverseries
                        </li>,
                        <li
                          className="color4"
                        >
                          And many more
                        </li>,
                      ],
                    },
                    "ref": null,
                    "rendered": Array [
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "To do list",
                          "className": "color1",
                        },
                        "ref": null,
                        "rendered": "To do list",
                        "type": "li",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "Aniverseries",
                          "className": "color2",
                        },
                        "ref": null,
                        "rendered": "Aniverseries",
                        "type": "li",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "And many more",
                          "className": "color4",
                        },
                        "ref": null,
                        "rendered": "And many more",
                        "type": "li",
                      },
                    ],
                    "type": "ul",
                  },
                ],
                "type": "div",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <div
                  className="form"
                >
                  <Connect(FormBox) />
                </div>,
                "className": "home-col",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Connect(FormBox) />,
                  "className": "form",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {},
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                "type": "div",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
